modules/installation-osp-deploying-bare-metal-machines.adoc

```diff
@@ -21,7 +21,8 @@ Bare-metal compute machines are not supported on clusters that use Kuryr.
 [NOTE]
 ====
 Be sure that your `install-config.yaml` file reflects whether the {rh-openstack} network that you use for bare metal workers supports floating IP addresses or not.
-
+When installing a OpenShift cluster with bare metal workers on {rh-openstack} 16.1 with OVN, the cluster is unable to the create worker nodes due to a link:https://bugzilla.redhat.com/show_bug.cgi?id=2033953[known issue]. The link:https://bugzilla.redhat.com/show_bug.cgi?id=2083120[same issue] has been fixed on 16.2, so it's recommended to upgrade to that version to use bare metal workers.

```

Comment:
> When installing a OpenShift cluster with bare metal workers on {rh-openstack} 16.1 with OVN, the cluster is unable to the create worker nodes due to a link:https://bugzilla.redhat.com/show_bug.cgi?id=2033953[known issue].

Note that the bare-metal workers setup worked up to OSP version [16.1.6](http://rhos-qe-mirror-tlv.usersys.redhat.com/rcm-guest/puddles/OpenStack/16.1-RHEL-8/z6/COMPOSE_ID) (OVN version ovn2.13-20.12.0-104.el8fdp.x86_64), and the regression is since OSP version 16.1.7 (OVN version ovn2.13-20.12.0-192.el8fdp.x86_64) - BMWs are not able to access the metadata service when running OSP 16.1.7 version or higher.